test/http-invocation.test.js

```diff
@@ -193,6 +193,146 @@ describe('HttpInvocation', function() {
       inv.transformResponse(res, body, cb);
     }
   });
+
+  describe('createRequest', function() {
+    it('should create a simple request', function() {
```

Comment:
@horiaradu : Please change all the `it` cases to reflect **imperative mood** as described in the Code style guidelines [here](http://loopback.io/doc/en/contrib/style-guide.html#test-naming).